### DISCUSSION TITLE
Error on deployment

### DISCUSSION THREAD

**Discussion by Kar1mm** [vercel/community#1943] [Help] [answered]
I need help![image](https://user-images.githubusercontent.com/80357871/229355340-fe24faf1-788b-4e9a-8381-ec674d6d9ec1.jpeg)
![image](https://user-images.githubusercontent.com/80357871/229355355-b5b4ae73-1fd4-412c-ab94-5a32cfe288c7.png)

**Comment by swarnava** [comment, marked as answer]
This looks like a depency error, you can include `npm install --legacy-peer-deps` to fix the issue. The same problem should appear on your local machine if you use same Node version to build the project.
